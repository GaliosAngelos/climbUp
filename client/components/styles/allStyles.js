import { StyleSheet } from "react-native";

const styles = StyleSheet.create({
  h1: {
    fontSize: 30,
    fontWeight: "bold",
    textAlign: "left",
  },

  h2: {
    fontSize: 24,
    fontWeight: "bold",
  },

  h3: {
    fontSize: 19,
    fontWeight: "bold",
  },

  text: {
    fontSize: 17,
    fontWeight: "medium",
  },

  head: {
    height: 130,
    justifyContent: "center",
    marginTop: 20,
  },

  textinput: {
    borderColor: "#B9B9B9",
    borderWidth: 3,
    borderRadius: 16,
    padding: 9,
    marginBottom: 20,
  },

  borderBox: {
    backgroundColor: "lightgrey",
    borderColor: "lightgrey",
    borderWidth: 4,
    borderRadius: 20,
    marginBottom: 20,
    paddingLeft: 20,
    paddingVertical: 15,
  },

  borderBoxExtended: {
    backgroundColor: "lightgrey",
    borderColor: "lightgrey",
    borderWidth: 4,
    borderTopLeftRadius: 20,
    borderTopRightRadius: 20,
    paddingLeft: 20,
    paddingVertical: 15,
  },

  routeExtension: {
    backgroundColor: "lightgrey",
    borderBottomLeftRadius: 20,
    borderBottomRightRadius: 20,
    borderColor: "lightgrey",
    borderEndWidth: 4,
    borderStartWidth: 4,
    borderBottomWidth: 4,
    height: 188,
    marginBottom: 20,
  },

  buttonlarge: {
    backgroundColor: "lightgrey",
    paddingVertical: 9,
    paddingHorizontal: 20,
    marginVertical: 10,
    borderRadius: 12,
    justifyContent: "center",
  },

  buttonlargetext: {
    fontSize: 20,
    fontWeight: "bold",
    textAlign: "center",
  },

  buttonmedium: {
    backgroundColor: "lightgrey",
    paddingVertical: 9,
    paddingHorizontal: 20,
    marginVertical: 10,
    borderRadius: 12,
    justifyContent: "center",
    flex: 1, // Sorgt dafür, dass der Button flexibel ist
    marginHorizontal: 5, // Gibt etwas Abstand zwischen den Buttons
  },

  buttonsmall: {
    marginBottom: "3%",
  },

  buttonsmalltext: {
    textDecorationLine: "underline",
    fontSize: 17,
    fontWeight: "bold",
    textAlign: "right",
    color: "#339C54",
  },

  selectedButton: {
    backgroundColor: "#484848",
  },

  centeredView: {
    flex: 1,
    justifyContent: "center",
    alignItems: "center",
    marginTop: 22,
  },
  modalView: {
    margin: 20,
    backgroundColor: "white",
    borderRadius: 20,
    padding: 20,
    alignItems: "center",
    elevation: 5,
    height: 300,
  },
  box: {
    width: 50, // von 60 auf 50 reduziert, um die Box kleiner zu machen
    height: 50, // von 60 auf 50 reduziert, um die Box kleiner zu machen
    justifyContent: "center",
    alignItems: "center",
    margin: 3,
    borderWidth: 1,
    borderColor: "#ddd",
    borderRadius: 10, // Runde Ecken hinzugefügt
  },
});

export default styles;
