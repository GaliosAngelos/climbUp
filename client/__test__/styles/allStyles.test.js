import React from "react";
import { render } from "@testing-library/react-native";
import styles from "../../components/styles/allStyles";
describe('Styles', () => {
  it('should match snapshot', () => {
    expect(styles).toMatchSnapshot();
  });
});