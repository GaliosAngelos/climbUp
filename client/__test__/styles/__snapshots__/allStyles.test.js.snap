// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styles should match snapshot 1`] = `
{
  "borderBox": {
    "backgroundColor": "lightgrey",
    "borderColor": "lightgrey",
    "borderRadius": 20,
    "borderWidth": 4,
    "marginBottom": 20,
    "paddingLeft": 20,
    "paddingVertical": 15,
  },
  "borderBoxExtended": {
    "backgroundColor": "lightgrey",
    "borderColor": "lightgrey",
    "borderTopLeftRadius": 20,
    "borderTopRightRadius": 20,
    "borderWidth": 4,
    "paddingLeft": 20,
    "paddingVertical": 15,
  },
  "box": {
    "alignItems": "center",
    "borderColor": "#ddd",
    "borderRadius": 10,
    "borderWidth": 1,
    "height": 50,
    "justifyContent": "center",
    "margin": 3,
    "width": 50,
  },
  "buttonlarge": {
    "backgroundColor": "lightgrey",
    "borderRadius": 12,
    "justifyContent": "center",
    "marginVertical": 10,
    "paddingHorizontal": 20,
    "paddingVertical": 9,
  },
  "buttonlargetext": {
    "fontSize": 20,
    "fontWeight": "bold",
    "textAlign": "center",
  },
  "buttonmedium": {
    "backgroundColor": "lightgrey",
    "borderRadius": 12,
    "flex": 1,
    "justifyContent": "center",
    "marginHorizontal": 5,
    "marginVertical": 10,
    "paddingHorizontal": 20,
    "paddingVertical": 9,
  },
  "buttonsmall": {
    "marginBottom": "3%",
  },
  "buttonsmalltext": {
    "color": "#339C54",
    "fontSize": 17,
    "fontWeight": "bold",
    "textAlign": "right",
    "textDecorationLine": "underline",
  },
  "centeredView": {
    "alignItems": "center",
    "flex": 1,
    "justifyContent": "center",
    "marginTop": 22,
  },
  "h1": {
    "fontSize": 30,
    "fontWeight": "bold",
    "textAlign": "left",
  },
  "h2": {
    "fontSize": 24,
    "fontWeight": "bold",
  },
  "h3": {
    "fontSize": 19,
    "fontWeight": "bold",
  },
  "head": {
    "height": 130,
    "justifyContent": "center",
    "marginTop": 20,
  },
  "modalView": {
    "alignItems": "center",
    "backgroundColor": "white",
    "borderRadius": 20,
    "elevation": 5,
    "height": 300,
    "margin": 20,
    "padding": 20,
  },
  "routeExtension": {
    "backgroundColor": "lightgrey",
    "borderBottomLeftRadius": 20,
    "borderBottomRightRadius": 20,
    "borderBottomWidth": 4,
    "borderColor": "lightgrey",
    "borderEndWidth": 4,
    "borderStartWidth": 4,
    "height": 188,
    "marginBottom": 20,
  },
  "selectedButton": {
    "backgroundColor": "#484848",
  },
  "text": {
    "fontSize": 17,
    "fontWeight": "medium",
  },
  "textinput": {
    "borderColor": "#B9B9B9",
    "borderRadius": 16,
    "borderWidth": 3,
    "marginBottom": 20,
    "padding": 9,
  },
}
`;
